grpc-interface-support/test/integration/test_integration_python.py

```diff
@@ -66,22 +66,25 @@ def assert_python_package_generated(service_name: str):
 
 
 def test_pip_package_is_usable():
-    os.chdir(os.environ["SERVICE_CLIENT_ROOT"])
-    assert subprocess.check_call(["velocitas", "init", "-v"]) == 0
-
-    from seats_service_sdk.SeatsServiceClientFactory import SeatsServiceClientFactory
     from velocitas_sdk.base import Middleware, ServiceLocator
 
-    class TestServiceLocator(ServiceLocator):
+    class TestClientServiceLocator(ServiceLocator):
+        def get_service_location(self, service_name: str) -> str:
+            return f"{service_name}@127.0.0.1:1234"  # noqa: E231
+
+        def get_metadata(self, service_name: Optional[str] = None):
+            pass
+
+    class TestServerServiceLocator(ServiceLocator):
```

Comment:
do we really need 2 different locators?

Reply:
yes I know, but do we really need different locators for client and servers? I would assume that a locator does not really care if it is a client or a server.

Reply:
If the service location is different, then I think yes.